### ISSUE TITLE
[Bug]: Multi Smelter breaking over night

### ISSUE THREAD

**Issue by Stwowar** [ThePansmith/Monifactory#944]
### Issue Summary

twice now my multi smelter was working fine and the next morning it looks like this: see 1st SS
![2024-10-15_12 03 36](https://github.com/user-attachments/assets/9f8f35a6-2b77-4294-9057-f552ac19caca)
, but as soon as i look at the Multi Smelter block it updates then changes to say: see 2nd SS
![2024-10-15_12 03 43](https://github.com/user-attachments/assets/10f731c6-de4e-4869-af28-c9d0f4d2531c)
, and then a split second later it says, see 3rd SS
![2024-10-15_12 03 45](https://github.com/user-attachments/assets/42820d92-7941-4697-b735-727d893ed9eb)
, and then is working fine, and so yes i had to be very quick in taking the 2nd SS.

TLDR: Just looking at it fixes it without any other interaction

### Reproduction steps

idk how to recreate it, from the last 3 mornings this has happened twice, and as far as i know there was no restarts or anything

EDIT: it has come to my knowledge there have been server restarts nightly, so that actually may be the cause, but i suppose still worth noting the bug

**Comment by Stwowar** [comment]
happened again today, though this time the block texture wasn't invisible, but looking at it immediately fixed it

**Comment by ThePansmith** [comment]
Likely a GTM issue, report it to them
